common/views/themes/base/wellcome-normalize.ts

```diff
@@ -0,0 +1,32 @@
+import { themeValues } from '../config';
+
+export const wellcomeNormalize = `
```

Comment:
Any reason to not just have normalize?

Reply:
we also have normalize as a vendor asset (still in scss at this point), but these were our own additions to that. I think keeping the two separate probably makes sense?

Reply:
Makes sense, could call it `extendedNormalize` or something, but nitpicking here.

Reply:
Assuming we could just call this normalise, pull in the vendor version and extend it?